src/modeler/AutoRest.Swagger/Validation/TrackedResourceGetOperationValidation.cs

```diff
@@ -24,6 +24,11 @@ public class TrackedResourceGetOperationValidation : TypedRule<Dictionary<string
         public override ValidationCategory ValidationCategory => ValidationCategory.RPCViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.SDKImpactingChanges;
```

Comment:
I believe this would be ServiceImpacting if the get operation is not implemented by the service.